include/wlr/types/wlr_region.h

```diff
@@ -1,17 +1,31 @@
 /*
- * This an unstable interface of wlroots. No guarantees are made regarding the
- * future consistency of this API.
+ * This is a stable interface of wlroots. Future changes will be limited to:
+ *
+ * - New functions
+ * - New struct members
+ * - New enum members
+ *
+ * Note that wlroots does not make an ABI compatibility promise - in the future,
+ * the layout and size of structs used by wlroots may change, requiring code
+ * depending on this header to be recompiled (but not edited).
+ *
+ * Breaking changes are announced by email and follow a 1-year deprecation
+ * schedule. Send an email to ~sircmpwn/wlroots-announce+subscribe@lists.sr.ht
```

Comment:
Should we actively use that mailing list? It seems unused

Reply:
I'll put in my vote for a mailing list where release notes are copied to alongside the existing GitHub atom feed.

Reply:
My take: https://github.com/swaywm/wlroots/pull/3016

Reply:
Seems fair. An atom feed is good enough.